docs/source/contribution/technical_steering_committee.md

```diff
@@ -1,5 +1,7 @@
 # Join the Technical Steering Committee
 
+In the Kedro project's latest iteration it is an incubating project within [LF AI & Data](https://lfaidata.foundation/).
+
 The term "Technical Steering Committee" (TSC) describes the group of Kedro maintainers.
 
 The TSC is responsible for the project's future development; you can read about our duties in our [Technical Charter](https://github.com/kedro-org/kedro/blob/main/kedro_technical_charter.pdf). We accept new members into the TSC to fuel Kedro's continued development. We list [Kedro's current and past maintainers](https://github.com/kedro-org/kedro/blob/main/README.md#the-humans-behind-kedro) on GitHub.
```

Comment:
This part will also need to be updated: "We list [Kedro's current and past maintainers](https://github.com/kedro-org/kedro/blob/main/README.md#the-humans-behind-kedro) on GitHub."